easybuild/easyblocks/t/trinity.py

```diff
@@ -261,12 +264,16 @@ def install_step(self):
             fn = "Makefile"
             for line in fileinput.input(fn, inplace=1, backup='.orig.eb'):
 
-                line = re.sub(r'^(INCHWORM_CONFIGURE_FLAGS\s*=\s*).*$', r'\1%s' % inchworm_flags, line)
-                line = re.sub(r'^(CHRYSALIS_MAKE_FLAGS\s*=\s*).*$', r'\1%s' % chrysalis_flags, line)
-                line = re.sub(r'(/rsem && \$\(MAKE\))\s*$',
-                              r'\1 CC=%s CXX="%s %s" CFLAGS_EXTRA="%s"\n' % (cc, cxx, lib_flags, lib_flags), line)
-                line = re.sub(r'(/fastool && \$\(MAKE\))\s*$',
-                              r'\1 CC="%s -std=c99" CFLAGS="%s ${CFLAGS}"\n' % (cc, lib_flags), line)
+                if LooseVersion(self.version) > "2" and LooseVersion(self.version) < "3":
+                    line = re.sub(r'^( INCHWORM_CONFIGURE_FLAGS\s*=\s*).*$', r'\1%s' % inchworm_flags, line)
+                    line = re.sub(r'^( CHRYSALIS_MAKE_FLAGS\s*=\s*).*$', r'\1%s' % chrysalis_flags, line)
+                else:
+                    line = re.sub(r'^(INCHWORM_CONFIGURE_FLAGS\s*=\s*).*$', r'\1%s' % inchworm_flags, line)
+                    line = re.sub(r'^(CHRYSALIS_MAKE_FLAGS\s*=\s*).*$', r'\1%s' % chrysalis_flags, line)
+                    line = re.sub(r'(/rsem && \$\(MAKE\))\s*$',
+                                  r'\1 CC=%s CXX="%s %s" CFLAGS_EXTRA="%s"\n' % (cc, cxx, lib_flags, lib_flags), line)
+                    line = re.sub(r'(/fastool && \$\(MAKE\))\s*$',
+                                  r'\1 CC="%s -std=c99" CFLAGS="%s ${CFLAGS}"\n' % (cc, lib_flags), line)
```

Comment:
since we're touching this, it would be better to switch to using `apply_regex_substitutions`, are you up for that @hajgato?

cfr. #725, PR #685 is an example